### PR TITLE
Accessibility : font size set to default 16px as mentioned in the accessibility bug WI: W-11149812

### PR DESCRIPTION
**Pull request by ssahay87**
Accessibility bug : the default font is set to 13px as of now, as per the guidelines setting it to 16px.
Adding screenshots for review
Screenshot 2023-11-27 at 3 33 05 PM
Screenshot 2023-11-27 at 3 33 20 PM
Screenshot 2023-11-27 at 8 24 55 PM

### CODE REVIEW

**Comment by ankurnigamsf** [comment]
Looks good to me

**Review by ankurnigamsf — DISMISSED** [review]

**Comment by ssahay87** [comment]
@jstvz I checked with accessibility team, we need 16 px as minimum for dektop site as well,  you can find the reference [here](https://salesforce-internal.slack.com/archives/C022Q6STSHG/p1702440232106799), cc- @ankurnigamsf @jkasturi-sf

File: src/js/components/products/list.tsx [hunk 1]
```diff
@@ -206,12 +207,12 @@ const ProductsList = () => {
       <>
         <Header history={history} />
         <PageHeader />
-        <div className="slds-p-around_x-large">
+        <div className="slds-text-heading_small slds-p-around_x-large">
           {window.GLOBALS.SITE?.welcome_text ? (
             // These messages are pre-cleaned by the API
             <div
               className="markdown
-                slds-p-bottom_medium
```
Comment (jstvz) [inline src/js/components/products/list.tsx:214]: Was deleting this intentional?

File: src/js/components/products/list.tsx [hunk 2]
```diff
@@ -206,12 +207,12 @@ const ProductsList = () => {
...
+                slds-text-heading_small
                 slds-text-longform
```
Comment (jstvz) [inline src/js/components/products/list.tsx:216]: Don't these two helper classes conflict with each other?
```suggestion
                slds-text-heading_small
```

**Review by jstvz — DISMISSED** [review]
Still confused about why SLDS isn't scaling to 16px on mobile as documented, but this works for now.

File: src/js/components/products/list.tsx (hunk 1 above)
Comment (ssahay87) [reply src/js/components/products/list.tsx:214]: reverted it

File: src/js/components/products/list.tsx (hunk 2 above)
Comment (ssahay87) [reply src/js/components/products/list.tsx]: removed longform

**Review by ankurnigamsf — DISMISSED** [review]
LGTM

**Review by jkasturi-sf — APPROVED** [review]
Making 16px as default font size

**Pull request merged**